source/platforms/coiled.md

```diff
@@ -82,7 +82,7 @@ We can also connect a Dask client to see that information for the workers too.
 ```python
 from dask.distributed import Client
 
-client = Client
+client = Client(cluster)
```

Comment:
Coming around to this PR now as I'm going through the Coiled deployment process - is this example meant to be specific to the Jupyter server case, or more an illustration of how to connect a Dask client generally?

Depending on the answer, might be worth clarifying here that (the screenshot below seems to imply) we can just use `Client()` from the Jupyter server to automatically connect to the spawned cluster

Reply:
That's a good point. The correct fix here would've just been `Client()` because this was referring to using it within the Jupyter session. @charlesbluca would you mind making a follow-up PR here?

Reply:
Yeah sure! Fixed in https://github.com/rapidsai/deployment/pull/473